npm-shrinkwrap.json

```diff
@@ -1,8 +1,11686 @@
 {
   "name": "fsh-sushi",
   "version": "1.3.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
It looks like you are using NPM 7.  We attempted to move to NPM 7 previously (I forget exactly when), but ended up having to keep it on NPM 6 due to some compatibility issues (which I don't currently remember).  It's probably worth evaluating again, but maybe not as part of this PR.

Since you did not actually change any dependencies, can we just revert this file back to the version on `master` for now?